deparse.c

```diff
@@ -1845,6 +1853,60 @@ sqlite_deparse_range_tbl_ref(StringInfo buf, PlannerInfo *root, RelOptInfo *fore
 										 params_list);
 }
 
+/*
+ * Add a RETURNING clause, if needed, to an INSERT/UPDATE/DELETE.
+ */
+static void
+sqlite_deparseReturningList(StringInfo buf, PlannerInfo *root,
+							Index rtindex, Relation rel,
+							bool trig_after_row,
+							List *withCheckOptionList,
+							List *returningList,
+							List **retrieved_attrs)
+{
+	Bitmapset  *attrs_used = NULL;
+
+	elog(DEBUG3, "sqlite_fdw : %s", __func__);
+	if (trig_after_row)
```

Comment:
Could you add test case to cover this condition?

Reply:
@lamdn1409 , I don't understand how can I test this `trig_after_row` flag. What this is about in our context?